INFO:::2015-08-14 13:23:16,665:::konfig_klase:::__init__:::LOC:31:::Lista defaultnih tocaka za umjeravanje, tocke=
INFO:::2015-08-14 13:23:16,665:::konfig_klase:::__init__:::LOC:52:::Lista defaultnih tocaka za provjeru konvertera, tocke=
INFO:::2015-08-14 13:23:16,665:::pomocne_funkcije:::pripremi_mape_postaja_i_uredjaja:::LOC:222:::Sastavljanje mapa postaja i uredjaja
INFO:::2015-08-14 13:23:16,681:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:16,697:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/drzavna_mreza/postaje HTTP/1.1" 200 4307
INFO:::2015-08-14 13:23:16,728:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:16,790:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj HTTP/1.1" 200 None
INFO:::2015-08-14 13:23:18,849:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:18,849:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A08117 HTTP/1.1" 200 1703
INFO:::2015-08-14 13:23:18,912:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:18,927:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A08117/lokacija HTTP/1.1" 200 556
INFO:::2015-08-14 13:23:18,943:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:18,959:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A08118 HTTP/1.1" 200 1703
INFO:::2015-08-14 13:23:19,021:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,021:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A08118/lokacija HTTP/1.1" 200 571
INFO:::2015-08-14 13:23:19,037:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,037:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A08119 HTTP/1.1" 200 1703
INFO:::2015-08-14 13:23:19,099:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,115:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A08119/lokacija HTTP/1.1" 200 566
INFO:::2015-08-14 13:23:19,130:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,130:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A08120 HTTP/1.1" 200 1703
INFO:::2015-08-14 13:23:19,193:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,193:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A08120/lokacija HTTP/1.1" 200 559
INFO:::2015-08-14 13:23:19,224:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,224:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A09001 HTTP/1.1" 200 1703
INFO:::2015-08-14 13:23:19,286:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,286:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A09001/lokacija HTTP/1.1" 200 559
INFO:::2015-08-14 13:23:19,302:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,317:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A09002 HTTP/1.1" 200 1703
INFO:::2015-08-14 13:23:19,380:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,380:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A09002/lokacija HTTP/1.1" 200 556
INFO:::2015-08-14 13:23:19,395:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,395:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A09003 HTTP/1.1" 200 1703
INFO:::2015-08-14 13:23:19,458:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,473:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A09003/lokacija HTTP/1.1" 200 748
INFO:::2015-08-14 13:23:19,489:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,489:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A09004 HTTP/1.1" 200 1703
INFO:::2015-08-14 13:23:19,551:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,567:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/18A09004/lokacija HTTP/1.1" 200 557
INFO:::2015-08-14 13:23:19,583:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,583:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2024-E HTTP/1.1" 200 3667
INFO:::2015-08-14 13:23:19,583:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,598:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2024-E/lokacija HTTP/1.1" 200 556
INFO:::2015-08-14 13:23:19,614:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,614:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2026-E HTTP/1.1" 200 3667
INFO:::2015-08-14 13:23:19,614:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,629:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2026-E/lokacija HTTP/1.1" 200 566
INFO:::2015-08-14 13:23:19,645:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,645:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2074-E HTTP/1.1" 200 3600
INFO:::2015-08-14 13:23:19,645:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,661:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2074-E/lokacija HTTP/1.1" 200 559
INFO:::2015-08-14 13:23:19,676:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,676:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2075-E HTTP/1.1" 200 3600
INFO:::2015-08-14 13:23:19,676:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,692:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2075-E/lokacija HTTP/1.1" 200 556
INFO:::2015-08-14 13:23:19,707:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,707:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2076-E HTTP/1.1" 200 3600
INFO:::2015-08-14 13:23:19,707:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,723:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2076-E/lokacija HTTP/1.1" 200 338
INFO:::2015-08-14 13:23:19,723:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,723:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2077-E HTTP/1.1" 200 3600
INFO:::2015-08-14 13:23:19,723:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,739:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2077-E/lokacija HTTP/1.1" 200 748
INFO:::2015-08-14 13:23:19,754:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,754:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2078-E HTTP/1.1" 200 3600
INFO:::2015-08-14 13:23:19,770:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,770:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2078-E/lokacija HTTP/1.1" 200 296
INFO:::2015-08-14 13:23:19,770:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,770:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2079-E HTTP/1.1" 200 3600
INFO:::2015-08-14 13:23:19,785:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,785:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2079-E/lokacija HTTP/1.1" 200 340
INFO:::2015-08-14 13:23:19,785:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,785:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2080-E HTTP/1.1" 200 3600
INFO:::2015-08-14 13:23:19,801:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,801:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2080-E/lokacija HTTP/1.1" 200 566
INFO:::2015-08-14 13:23:19,817:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,817:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2081-E HTTP/1.1" 200 3600
INFO:::2015-08-14 13:23:19,832:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,832:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2081-E/lokacija HTTP/1.1" 200 571
INFO:::2015-08-14 13:23:19,848:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,863:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2082-E HTTP/1.1" 200 3600
INFO:::2015-08-14 13:23:19,863:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,863:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2082-E/lokacija HTTP/1.1" 200 559
INFO:::2015-08-14 13:23:19,879:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,895:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2214-1-2 HTTP/1.1" 200 2325
INFO:::2015-08-14 13:23:19,895:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,895:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2214-1-2/lokacija HTTP/1.1" 200 557
INFO:::2015-08-14 13:23:19,910:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,926:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2215-1-2 HTTP/1.1" 200 2325
INFO:::2015-08-14 13:23:19,926:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,926:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2215-1-2/lokacija HTTP/1.1" 200 566
INFO:::2015-08-14 13:23:19,957:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,957:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2219-E HTTP/1.1" 200 3654
INFO:::2015-08-14 13:23:19,957:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,973:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2219-E/lokacija HTTP/1.1" 200 556
INFO:::2015-08-14 13:23:19,988:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:19,988:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2220-E HTTP/1.1" 200 3654
INFO:::2015-08-14 13:23:19,988:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,004:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2220-E/lokacija HTTP/1.1" 200 566
INFO:::2015-08-14 13:23:20,019:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,019:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2231 HTTP/1.1" 200 2317
INFO:::2015-08-14 13:23:20,035:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,035:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2231/lokacija HTTP/1.1" 200 556
INFO:::2015-08-14 13:23:20,051:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,066:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2232 HTTP/1.1" 200 2317
INFO:::2015-08-14 13:23:20,066:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,066:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2232/lokacija HTTP/1.1" 200 559
INFO:::2015-08-14 13:23:20,082:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,097:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2819-E HTTP/1.1" 200 4768
ERROR:::2015-08-14 13:23:20,097:::pomocne_funkcije:::get_uredjaj_info:::LOC:117:::Za uredjaj 2819-E nije definirana min granica za Opseg. Koristim default 0
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 113, in get_uredjaj_info
    minGranica = metoda.find('min').text
AttributeError: 'NoneType' object has no attribute 'text'
INFO:::2015-08-14 13:23:20,097:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,097:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2819-E/lokacija HTTP/1.1" 200 566
INFO:::2015-08-14 13:23:20,113:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,129:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2820-E HTTP/1.1" 200 4768
ERROR:::2015-08-14 13:23:20,129:::pomocne_funkcije:::get_uredjaj_info:::LOC:117:::Za uredjaj 2820-E nije definirana min granica za Opseg. Koristim default 0
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 113, in get_uredjaj_info
    minGranica = metoda.find('min').text
AttributeError: 'NoneType' object has no attribute 'text'
INFO:::2015-08-14 13:23:20,129:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,129:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2820-E/lokacija HTTP/1.1" 200 556
INFO:::2015-08-14 13:23:20,144:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,160:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2821-E HTTP/1.1" 200 4768
ERROR:::2015-08-14 13:23:20,160:::pomocne_funkcije:::get_uredjaj_info:::LOC:117:::Za uredjaj 2821-E nije definirana min granica za Opseg. Koristim default 0
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 113, in get_uredjaj_info
    minGranica = metoda.find('min').text
AttributeError: 'NoneType' object has no attribute 'text'
INFO:::2015-08-14 13:23:20,160:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,160:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/2821-E/lokacija HTTP/1.1" 200 571
INFO:::2015-08-14 13:23:20,191:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,191:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/CO_KT1 HTTP/1.1" 200 3252
INFO:::2015-08-14 13:23:20,191:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,191:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/CO_KT1/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,191:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/CO_KT1/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,207:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,207:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/CO_OS1 HTTP/1.1" 200 3252
INFO:::2015-08-14 13:23:20,207:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,222:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/CO_OS1/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,222:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/CO_OS1/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,222:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,222:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/CO_SK1 HTTP/1.1" 200 3252
INFO:::2015-08-14 13:23:20,238:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,238:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/CO_SK1/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,238:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/CO_SK1/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,238:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,238:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/CO_ZG1 HTTP/1.1" 200 3251
INFO:::2015-08-14 13:23:20,238:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,253:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/CO_ZG1/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,253:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/CO_ZG1/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,253:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,253:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/CO_ZG2 HTTP/1.1" 200 3251
INFO:::2015-08-14 13:23:20,253:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,269:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/CO_ZG2/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,269:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/CO_ZG2/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,269:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,269:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/CO_ZG3 HTTP/1.1" 200 3251
INFO:::2015-08-14 13:23:20,269:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,285:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/CO_ZG3/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,285:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/CO_ZG3/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,285:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,285:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/H2S_KT1 HTTP/1.1" 200 3244
INFO:::2015-08-14 13:23:20,285:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,300:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/H2S_KT1/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,300:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/H2S_KT1/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,300:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,300:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/H2S_SK1 HTTP/1.1" 200 3244
INFO:::2015-08-14 13:23:20,300:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,316:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/H2S_SK1/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,316:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/H2S_SK1/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,316:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,316:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/KT1_PM10 HTTP/1.1" 200 445
INFO:::2015-08-14 13:23:20,316:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,331:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/KT1_PM10/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,331:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/KT1_PM10/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,331:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,331:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/NH3_KT1 HTTP/1.1" 200 3612
ERROR:::2015-08-14 13:23:20,331:::pomocne_funkcije:::get_uredjaj_info:::LOC:117:::Za uredjaj NH3_KT1 nije definirana min granica za Opseg. Koristim default 0
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 113, in get_uredjaj_info
    minGranica = metoda.find('min').text
AttributeError: 'NoneType' object has no attribute 'text'
INFO:::2015-08-14 13:23:20,331:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,347:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/NH3_KT1/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,347:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/NH3_KT1/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,347:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,347:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/NO2_OS1 HTTP/1.1" 200 3607
ERROR:::2015-08-14 13:23:20,347:::pomocne_funkcije:::get_uredjaj_info:::LOC:117:::Za uredjaj NO2_OS1 nije definirana min granica za Opseg. Koristim default 0
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 113, in get_uredjaj_info
    minGranica = metoda.find('min').text
AttributeError: 'NoneType' object has no attribute 'text'
INFO:::2015-08-14 13:23:20,347:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,363:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/NO2_OS1/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,363:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/NO2_OS1/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,363:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,363:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/NO2_SK1 HTTP/1.1" 200 3607
ERROR:::2015-08-14 13:23:20,363:::pomocne_funkcije:::get_uredjaj_info:::LOC:117:::Za uredjaj NO2_SK1 nije definirana min granica za Opseg. Koristim default 0
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 113, in get_uredjaj_info
    minGranica = metoda.find('min').text
AttributeError: 'NoneType' object has no attribute 'text'
INFO:::2015-08-14 13:23:20,363:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,378:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/NO2_SK1/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,378:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/NO2_SK1/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,378:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,378:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/NO2_ZG1 HTTP/1.1" 200 3606
ERROR:::2015-08-14 13:23:20,378:::pomocne_funkcije:::get_uredjaj_info:::LOC:117:::Za uredjaj NO2_ZG1 nije definirana min granica za Opseg. Koristim default 0
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 113, in get_uredjaj_info
    minGranica = metoda.find('min').text
AttributeError: 'NoneType' object has no attribute 'text'
INFO:::2015-08-14 13:23:20,378:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,378:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/NO2_ZG1/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,378:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/NO2_ZG1/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,394:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,394:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/NO2_ZG2 HTTP/1.1" 200 3606
ERROR:::2015-08-14 13:23:20,394:::pomocne_funkcije:::get_uredjaj_info:::LOC:117:::Za uredjaj NO2_ZG2 nije definirana min granica za Opseg. Koristim default 0
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 113, in get_uredjaj_info
    minGranica = metoda.find('min').text
AttributeError: 'NoneType' object has no attribute 'text'
INFO:::2015-08-14 13:23:20,394:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,394:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/NO2_ZG2/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,394:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/NO2_ZG2/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,409:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,409:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/NO2_ZG3 HTTP/1.1" 200 3606
ERROR:::2015-08-14 13:23:20,409:::pomocne_funkcije:::get_uredjaj_info:::LOC:117:::Za uredjaj NO2_ZG3 nije definirana min granica za Opseg. Koristim default 0
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 113, in get_uredjaj_info
    minGranica = metoda.find('min').text
AttributeError: 'NoneType' object has no attribute 'text'
INFO:::2015-08-14 13:23:20,409:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,409:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/NO2_ZG3/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,409:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/NO2_ZG3/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,409:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,425:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/O3_ZG3 HTTP/1.1" 200 3236
INFO:::2015-08-14 13:23:20,425:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,425:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/O3_ZG3/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,425:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/O3_ZG3/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,425:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,441:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/OS1_BTX HTTP/1.1" 200 535
INFO:::2015-08-14 13:23:20,441:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,441:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/OS1_BTX/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,441:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/OS1_BTX/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,441:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,456:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/OS1_PM10 HTTP/1.1" 200 445
INFO:::2015-08-14 13:23:20,456:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,456:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/OS1_PM10/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,456:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/OS1_PM10/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,456:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,456:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI1_BTX HTTP/1.1" 200 535
INFO:::2015-08-14 13:23:20,472:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,472:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI1_BTX/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,472:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/RI1_BTX/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,472:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,472:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI1_CO HTTP/1.1" 200 3252
INFO:::2015-08-14 13:23:20,487:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,487:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI1_CO/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,487:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/RI1_CO/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,487:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,487:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI1_H2S HTTP/1.1" 200 3244
INFO:::2015-08-14 13:23:20,503:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,503:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI1_H2S/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,503:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/RI1_H2S/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,503:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,503:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI1_NO2 HTTP/1.1" 200 3607
ERROR:::2015-08-14 13:23:20,503:::pomocne_funkcije:::get_uredjaj_info:::LOC:117:::Za uredjaj RI1_NO2 nije definirana min granica za Opseg. Koristim default 0
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 113, in get_uredjaj_info
    minGranica = metoda.find('min').text
AttributeError: 'NoneType' object has no attribute 'text'
INFO:::2015-08-14 13:23:20,519:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,519:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI1_NO2/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,519:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/RI1_NO2/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,519:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,519:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI1_PM10 HTTP/1.1" 200 445
INFO:::2015-08-14 13:23:20,519:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,534:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI1_PM10/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,534:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/RI1_PM10/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,534:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,534:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI1_SO2 HTTP/1.1" 200 3239
INFO:::2015-08-14 13:23:20,534:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,550:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI1_SO2/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,550:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/RI1_SO2/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,550:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,550:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI2_CO HTTP/1.1" 200 3252
INFO:::2015-08-14 13:23:20,550:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,565:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI2_CO/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,565:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/RI2_CO/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,565:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,565:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI2_NO2 HTTP/1.1" 200 3607
ERROR:::2015-08-14 13:23:20,565:::pomocne_funkcije:::get_uredjaj_info:::LOC:117:::Za uredjaj RI2_NO2 nije definirana min granica za Opseg. Koristim default 0
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 113, in get_uredjaj_info
    minGranica = metoda.find('min').text
AttributeError: 'NoneType' object has no attribute 'text'
INFO:::2015-08-14 13:23:20,565:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,581:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI2_NO2/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,581:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/RI2_NO2/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,581:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,581:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI2_O3 HTTP/1.1" 200 3236
INFO:::2015-08-14 13:23:20,581:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,597:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI2_O3/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,597:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/RI2_O3/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,597:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,597:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI2_PM10 HTTP/1.1" 200 445
INFO:::2015-08-14 13:23:20,597:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,612:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI2_PM10/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,612:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/RI2_PM10/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,612:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,612:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI2_SO2 HTTP/1.1" 200 3239
INFO:::2015-08-14 13:23:20,612:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,628:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/RI2_SO2/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,628:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/RI2_SO2/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,628:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,628:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB1_BTX HTTP/1.1" 200 535
INFO:::2015-08-14 13:23:20,628:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,628:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB1_BTX/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,643:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/SB1_BTX/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,643:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,643:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB1_H2S HTTP/1.1" 200 3244
INFO:::2015-08-14 13:23:20,643:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,643:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB1_H2S/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,643:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/SB1_H2S/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,659:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,659:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB1_NO2 HTTP/1.1" 200 3607
ERROR:::2015-08-14 13:23:20,659:::pomocne_funkcije:::get_uredjaj_info:::LOC:117:::Za uredjaj SB1_NO2 nije definirana min granica za Opseg. Koristim default 0
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 113, in get_uredjaj_info
    minGranica = metoda.find('min').text
AttributeError: 'NoneType' object has no attribute 'text'
INFO:::2015-08-14 13:23:20,659:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,675:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB1_NO2/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,675:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/SB1_NO2/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,675:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,675:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB1_O3 HTTP/1.1" 200 3236
INFO:::2015-08-14 13:23:20,675:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,690:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB1_O3/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,690:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/SB1_O3/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,690:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,690:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB1_PM25 HTTP/1.1" 200 448
INFO:::2015-08-14 13:23:20,690:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,706:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB1_PM25/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,706:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/SB1_PM25/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,706:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,706:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB1_SO2 HTTP/1.1" 200 3239
INFO:::2015-08-14 13:23:20,706:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,721:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB1_SO2/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,721:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/SB1_SO2/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,721:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,721:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB2_BTX HTTP/1.1" 200 584
INFO:::2015-08-14 13:23:20,721:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,737:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB2_BTX/lokacija HTTP/1.1" 200 599
INFO:::2015-08-14 13:23:20,753:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,768:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB2_CO HTTP/1.1" 200 3290
INFO:::2015-08-14 13:23:20,768:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,768:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB2_CO/lokacija HTTP/1.1" 200 599
INFO:::2015-08-14 13:23:20,784:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,799:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB2_H2S HTTP/1.1" 200 3277
INFO:::2015-08-14 13:23:20,799:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,799:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB2_H2S/lokacija HTTP/1.1" 200 599
INFO:::2015-08-14 13:23:20,815:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,831:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB2_PM HTTP/1.1" 200 491
INFO:::2015-08-14 13:23:20,831:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,831:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB2_PM/lokacija HTTP/1.1" 200 599
INFO:::2015-08-14 13:23:20,846:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,862:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB2_SO2 HTTP/1.1" 200 3277
INFO:::2015-08-14 13:23:20,862:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,877:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SB2_SO2/lokacija HTTP/1.1" 200 599
INFO:::2015-08-14 13:23:20,893:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,893:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SK1_BTX HTTP/1.1" 200 532
INFO:::2015-08-14 13:23:20,909:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,909:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SK1_BTX/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,909:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/SK1_BTX/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,909:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,909:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SK1_PM10 HTTP/1.1" 200 445
INFO:::2015-08-14 13:23:20,924:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,924:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SK1_PM10/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,924:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/SK1_PM10/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,924:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,924:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SO2_KT1 HTTP/1.1" 200 3239
INFO:::2015-08-14 13:23:20,924:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,940:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SO2_KT1/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,940:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/SO2_KT1/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,940:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,940:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SO2_OS1 HTTP/1.1" 200 3239
INFO:::2015-08-14 13:23:20,940:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,955:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SO2_OS1/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,955:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/SO2_OS1/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,955:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,955:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SO2_SK1 HTTP/1.1" 200 3239
INFO:::2015-08-14 13:23:20,955:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,971:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SO2_SK1/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:20,971:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/SO2_SK1/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:20,971:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:20,971:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SO2_ZG1 HTTP/1.1" 200 3238
INFO:::2015-08-14 13:23:20,971:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,018:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SO2_ZG1/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:21,018:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/SO2_ZG1/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:21,018:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,018:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SO2_ZG2 HTTP/1.1" 200 3238
INFO:::2015-08-14 13:23:21,033:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,033:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SO2_ZG2/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:21,033:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/SO2_ZG2/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:21,033:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,033:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SO2_ZG3 HTTP/1.1" 200 3238
INFO:::2015-08-14 13:23:21,049:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,049:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/SO2_ZG3/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:21,049:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/SO2_ZG3/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:21,049:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,065:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/ZG1_BTX HTTP/1.1" 200 535
INFO:::2015-08-14 13:23:21,065:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,065:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/ZG1_BTX/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:21,065:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/ZG1_BTX/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:21,065:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,065:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/ZG1_PM10 HTTP/1.1" 200 445
INFO:::2015-08-14 13:23:21,065:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,080:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/ZG1_PM10/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:21,080:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/ZG1_PM10/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:21,080:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,080:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/ZG1_PM10g HTTP/1.1" 200 445
INFO:::2015-08-14 13:23:21,080:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,096:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/ZG1_PM10g/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:21,096:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/ZG1_PM10g/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:21,096:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,096:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/ZG2_PM10 HTTP/1.1" 200 445
INFO:::2015-08-14 13:23:21,096:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,096:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/ZG2_PM10/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:21,111:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/ZG2_PM10/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:21,111:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,111:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/ZG3_PM10 HTTP/1.1" 200 445
INFO:::2015-08-14 13:23:21,111:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,111:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/ZG3_PM10/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:21,111:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/ZG3_PM10/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
INFO:::2015-08-14 13:23:21,111:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,127:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/ZG3_PM10g HTTP/1.1" 200 445
INFO:::2015-08-14 13:23:21,127:::connectionpool:::_new_conn:::LOC:203:::Starting new HTTP connection (1): 172.20.0.178
DEBUG:::2015-08-14 13:23:21,127:::connectionpool:::_make_request:::LOC:383:::"GET /SKZ-war/webresources/uredjaj/ZG3_PM10g/lokacija HTTP/1.1" 204 0
ERROR:::2015-08-14 13:23:21,127:::pomocne_funkcije:::get_lokaciju_uredjaja:::LOC:178:::Pogreska kod trazenja lokacije uredjaja url=http://172.20.0.178:8080/SKZ-war/webresources/uredjaj/ZG3_PM10g/lokacija
Traceback (most recent call last):
  File "C:\Users\DHMZ-Milic\Documents\GitHub\umjeravanje\app\model\pomocne_funkcije.py", line 169, in get_lokaciju_uredjaja
    root = ET.fromstring(r.text)
  File "C:\WinPython-64bit-3.4.3.1\python-3.4.3.amd64\lib\xml\etree\ElementTree.py", line 1326, in XML
    return parser.close()
  File "<string>", line None
xml.etree.ElementTree.ParseError: no element found: line 1, column 0
DEBUG:::2015-08-14 13:23:21,127:::kalkulator:::__init__:::LOC:19:::start inicijalizacije RacunUmjeravanja
DEBUG:::2015-08-14 13:23:21,127:::kalkulator:::reset_results:::LOC:120:::All result members reset to np.NaN
DEBUG:::2015-08-14 13:23:21,127:::kalkulator:::__init__:::LOC:31:::kraj inicijalizacije RacunUmjeravanja
DEBUG:::2015-08-14 13:23:21,143:::kalkulator:::__init__:::LOC:561:::start inicijalizacije ProvjeraKonvertera
DEBUG:::2015-08-14 13:23:21,143:::kalkulator:::reset_results:::LOC:607:::All result members reset to "n/a"
DEBUG:::2015-08-14 13:23:21,143:::kalkulator:::__init__:::LOC:568:::kraj inicijalizacije ProvjeraKonvertera
